eng/code-quality-reports/src/main/java/com/azure/tools/revapi/transforms/AzureSdkAllowedExternalApis.java

```diff
@@ -75,12 +75,15 @@ private static ExternalApiStatus shouldExternalApiBeIgnored(TypeElement element)
                     || "core.".regionMatches(0, className, 10, 5)
                     || "cosmos.".regionMatches(0, className, 10, 7)
                     || "data.schemaregistry.".regionMatches(0, className, 10, 20)
+                    || "data.appconfiguration.".regionMatches(0, className, 10, 22)
                     || "json.".regionMatches(0, className, 10, 5)
                     || "messaging.eventgrid.".regionMatches(0, className, 10, 20)
                     || "messaging.eventhubs.".regionMatches(0, className, 10, 20)
                     || "messaging.servicebus.".regionMatches(0, className, 10, 21)
                     || "resourcemanager.".regionMatches(0, className, 10, 16)
                     || "security.keyvault.".regionMatches(0, className, 10, 18)
+                    || "spring.cloud.config.".regionMatches(0, className, 10, 20)
+                    || "spring.cloud.feature.".regionMatches(0, className, 10, 21)
```

Comment:
I just noticed that we should use `spring.cloud.azure.config` and `spring.cloud.azure.feature` as the top package name.

Reply:
Is there a good reason for the name change at this point? And if so. We can do it in a second PR, before GA.

Reply:
Oh, I was wrong about that, `com.azure.spring.cloud` is correct.